rust/types/src/lib.rs

```diff
@@ -47,6 +47,7 @@ pub use tenant::*;
 pub use types::*;
 pub use where_parsing::*;
 
+#[allow(clippy::all)]
```

Comment:
this is needed to ignore a new warning in the prost-genenerated code--happens because we have a service with no methods and prost generates slightly unidiomatic code